integreat_cms/cms/migrations/0081_poi_appointment_url.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.20 on 2023-09-21 08:22
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    Add field for setting appointment link
+    """
+
+    dependencies = [
+        ("cms", "0080_alter_poi_category"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="poi",
+            name="appointment_url",
+            field=models.URLField(
+                blank=True,
+                help_text="Link to an external website where an appointment for this location can be made.",
+                max_length=500,
+                verbose_name="appointment link",
+            ),
+        ),
```

Comment:
```suggestion
        ),
        migrations.RunPython(set_default_appointment_only),
```
Since the two migrations relate to each other, I think we can move them to the same file.